### ISSUE TITLE
Single-line comments not picked up by SlevomatCodingStandard.Commenting.ForbiddenComments

### ISSUE THREAD

**Issue by Hornet-Wing** [slevomat/coding-standard#717]
Firstly, thanks for the great coding standard!

Would it be possible to use the `SlevomatCodingStandard.Commenting.ForbiddenComments` sniff to flag up single line comments too?

I'd like to use this sniff for left over development code such as `var_dump` or (in Laravel) `dd`.

Example:
```xml
<rule ref="SlevomatCodingStandard.Commenting.ForbiddenComments">
	<properties>
		<property name="forbiddenCommentPatterns" type="array">
			<element value="~var_dump~"  />
		</property>
	</properties>
</rule>
```

```php
class Foo {
	/** var_dump('This comment will be flagged') */

	// var_dump('This comment will not flagged')
}
```

I tried adding `T_COMMENT` to the registration array in the sniff but it looks like single-line comments are filtered out earlier. Am I correct?

**Comment by kukulich** [comment]
I think you should use `Squiz.PHP.CommentedOutCode` sniff.
